server.js

```diff
@@ -0,0 +1,12 @@
+const http = require('http');
+
+const server = http.createServer((req, res) => {
+    res.end(JSON.stringify({
+        host: "https://adb-309687753508875.15.azuredatabricks.net/",
+        headers: {
+            "Authorization": "Bearer dapi54a1fe8b566376754ecb9ae1d1e20761-2",
```

Comment:
a) why do you need this one?
b) why credential is posted in a clear text?
c) if you need a server, use the https://pkg.go.dev/net/http/httptest package or HTTP Fixture framework, no JavaScript is required in this repo